docs(self-managed): add download link for Docker Compose (#4878)
camunda/camunda-docs: docs/self-managed/setup/deploy/local/docker-compose.md

To start a complete instance of Camunda 8 Self-Managed environment locally:

1. Download the [Camunda 8 Docker Compose artifact](https://github.com/camunda/camunda-self-managed/releases/download/docker-compose-alpha/docker-compose-alpha.zip) and extract it.

> 2. Enter the extracted directory, and run the following command:

Reviewer: ```suggestion
2. Run the following command in the extracted directory:
```
Rewording suggestion 👍
